src/api/db/structure.sql

```diff
@@ -1130,7 +1138,7 @@ CREATE TABLE `staging_workflows` (
   `updated_at` datetime NOT NULL,
   PRIMARY KEY (`id`),
   UNIQUE KEY `index_staging_workflows_on_project_id` (`project_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
This looks wrong - and unrelated

Reply:
@eduardoj You must be using an old database.yml.

Reply:
Fixed.